src/test/fuzz/timedata.cpp

```diff
@@ -1,31 +0,0 @@
-// Copyright (c) 2020-2021 The Bitcoin Core developers
```

Comment:
Could you add a fuzz test for `TimeOffsets` to replace the timedata one?

Reply:
Can do, but given how simple the interface is I think it wouldn't be anything more than the below (untested)? Could e.g. add the `.Size()` method back but adding it just for fuzzing might not be ideal?

<details>
<summary>git diff on 55b476b015</summary>

```diff
diff --git a/src/Makefile.test.include b/src/Makefile.test.include
index c77056f29c..c20b531032 100644
--- a/src/Makefile.test.include
+++ b/src/Makefile.test.include
@@ -381,6 +381,7 @@ test_fuzz_fuzz_SOURCES = \
  test/fuzz/string.cpp \
  test/fuzz/strprintf.cpp \
  test/fuzz/system.cpp \
+ test/fuzz/timeoffsets.cpp \
  test/fuzz/torcontrol.cpp \
  test/fuzz/transaction.cpp \
  test/fuzz/tx_in.cpp \
diff --git a/src/test/fuzz/timeoffsets.cpp b/src/test/fuzz/timeoffsets.cpp
new file mode 100644
index 0000000000..f0d2a8f739
--- /dev/null
+++ b/src/test/fuzz/timeoffsets.cpp
@@ -0,0 +1,22 @@
+// Copyright (c) 2020-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <node/timeoffsets.h>
+
+#include <chrono>
+
+
+FUZZ_TARGET(timeoffsets)
+{
+    FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+    TimeOffsets offsets{};
+    while (fuzzed_data_provider.remaining_bytes() > 0) {
+        (void)offsets.Median();
+        offsets.Add(std::chrono::seconds{fuzzed_data_provider.ConsumeIntegral<int64_t>()});
+        offsets.WarnIfOutOfSync();
+    }
+}

```
</details>

Reply:
> but given how simple the interface is I think it wouldn't be anything more than the below

That's good enough. We just want the code to be under test, in case we have change it in the future.

Reply:
Done. I was a bit worried `WarnIfOutOfSync()` outputting to stderr would mess with the fuzzer (as [here](https://github.com/bitcoin/bitcoin/pull/29623/commits/48cc160d70e4d918f5e2e5440b0ab5891499f5cf)), but running it locally seems to work fine so far, not sure if the appropriate paths have been hit already though.

Resolving for now.